CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unversioned
 
 - Minor: Delete all but the last 5 crashdumps on application start. (#4392)
+- Minor: Added `/banid` command that allows banning by user ID. (#4411)
```

Comment:
```suggestion
- Minor: Added `/banid` command that allows banning by userID. (#4411)
```
I have a vague memory of some argument why mentions of userids should be `userID` and while I don't remember what we we decided on or even if it was for chatterino/pajbot, but I think no space is more correct